- skip firefox files until the extension actually works
- avoid downloading and shipping the firefox extension, users download it
  pending "Linux, BSD: Add BUNDLE_XPI option" https://github.com/web-eid/web-eid-app/pull/275

Index: src/app/CMakeLists.txt
--- src/app/CMakeLists.txt.orig
+++ src/app/CMakeLists.txt
@@ -163,15 +163,11 @@ elseif(APPLE)
     endif()
 else()
     include(GNUInstallDirs)
-    file(DOWNLOAD ${FIREFOX_URL} ${CMAKE_CURRENT_BINARY_DIR}/${FIREFOX_UUID}.xpi)
     set(WEBEID_PATH ${CMAKE_INSTALL_FULL_BINDIR}/web-eid)
     install(TARGETS web-eid DESTINATION ${CMAKE_INSTALL_BINDIR})
     if(EXISTS "/etc/debian_version")
         install(FILES ${CMAKE_CURRENT_BINARY_DIR}/eu.webeid.firefox.json
             DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/mozilla/native-messaging-hosts RENAME eu.webeid.json)
-    else()
-        install(FILES ${CMAKE_CURRENT_BINARY_DIR}/eu.webeid.firefox.json
-            DESTINATION ${CMAKE_INSTALL_LIBDIR}/mozilla/native-messaging-hosts RENAME eu.webeid.json)
     endif()
     install(FILES ${CMAKE_CURRENT_BINARY_DIR}/eu.webeid.json
         DESTINATION ${CMAKE_INSTALL_DATADIR}/web-eid)
@@ -179,8 +175,6 @@ else()
         DESTINATION ${CMAKE_INSTALL_DATADIR}/google-chrome/extensions)
     install(FILES ${CMAKE_SOURCE_DIR}/install/web-eid.desktop
         DESTINATION ${CMAKE_INSTALL_DATADIR}/applications)
-    install(FILES ${CMAKE_CURRENT_BINARY_DIR}/${FIREFOX_UUID}.xpi
-            DESTINATION ${CMAKE_INSTALL_DATADIR}/mozilla/extensions/{ec8030f7-c20a-464f-9b0e-13a3a9e97384})
     foreach(RES 16 32 128 256 512)
         install(
             FILES ${CMAKE_SOURCE_DIR}/install/appicon_${RES}.png
